suite-native/atoms/src/Pictogram/Pictogram.tsx

```diff
@@ -0,0 +1,97 @@
+import React from 'react';
+
+import { Icon, IconName } from '@suite-native/icons';
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+import { Color } from '@trezor/theme';
+
+import { Box } from '../Box';
+import { CriticalIconSvg } from './CriticalIconSvg';
+import { CriticalShapeSvg } from './CriticalShapeSvg';
+import { InfoIconSvg } from './InfoIconSvg';
+import { InfoShapeSvg } from './InfoShapeSvg';
+import { SuccessIconSvg } from './SuccessIconSvg';
+import { SuccessShapeSvg } from './SuccessShapeSvg';
+import { WarningIconSvg } from './WarningIconSvg';
+import { WarningShapeSvg } from './WarningShapeSvg';
+
+export type PictogramVariant = 'success' | 'info' | 'warning' | 'critical';
+
+type PictogramProps = {
+    variant: PictogramVariant;
+    icon?: IconName;
+};
+
+type PictogramConfig = {
+    ShapeSvg:
+        | typeof SuccessShapeSvg
+        | typeof InfoShapeSvg
+        | typeof WarningShapeSvg
+        | typeof CriticalShapeSvg;
+    IconSvg:
+        | typeof SuccessIconSvg
+        | typeof InfoIconSvg
+        | typeof WarningIconSvg
+        | typeof CriticalIconSvg;
+    iconColor: Color;
+    iconOffset: number;
+};
+
+const pictogramVariantsMap = {
+    success: {
+        ShapeSvg: SuccessShapeSvg,
+        IconSvg: SuccessIconSvg,
+        iconOffset: 0,
+        iconColor: 'iconPrimaryDefault',
+    },
+    info: {
+        ShapeSvg: InfoShapeSvg,
+        IconSvg: InfoIconSvg,
+        iconOffset: 0,
+        iconColor: 'iconAlertBlue',
+    },
+    warning: {
+        ShapeSvg: WarningShapeSvg,
+        IconSvg: WarningIconSvg,
+        iconOffset: 20,
+        iconColor: 'iconAlertYellow',
+    },
+    critical: {
+        ShapeSvg: CriticalShapeSvg,
+        IconSvg: CriticalIconSvg,
+        iconOffset: 0,
+        iconColor: 'iconAlertRed',
+    },
+} as const satisfies Record<PictogramVariant, PictogramConfig>;
+
+const pictogramContainerStyle = prepareNativeStyle(_ => ({
+    width: 112,
```

Comment:
use the global value here as well please.

Reply:
Same comment as above.